src/collective/privacy/zcml.py

```diff
@@ -1,13 +1,15 @@
 # -*- coding: utf-8 -*-
+from collective.privacy.interfaces import ILawfulBasis, IProcessingReason
```

Comment:
please split this in 2 lines.

Reply:
why code-analysis does not complain about this?

Reply:
I have just seen that setup.cfg has some lines ignoring some violations.

Reply:
I very much dislike how prescriptive some of the code analysis tools are by default, so on my projects I routinely disable some of the noisier checkers.